ETMovies/ETMovies/Models/Studio.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ETMovies.Models
+{
+
+	public class Studio
+	{
+		public int StudioID { get; set; }
```

Comment:
ID should be enough here, its not needed to specify again the name of the entity, we do this only when we use reference to another entity, like MovieID